common/src/main/java/com/microsoft/identity/common/internal/dto/PrimaryRefreshTokenRecord.java

```diff
@@ -0,0 +1,256 @@
+// Copyright (c) Microsoft Corporation.
+// All rights reserved.
+//
+// This code is licensed under the MIT License.
+//
+// Permission is hereby granted, free of charge, to any person obtaining a copy
+// of this software and associated documentation files(the "Software"), to deal
+// in the Software without restriction, including without limitation the rights
+// to use, copy, modify, merge, publish, distribute, sublicense, and / or sell
+// copies of the Software, and to permit persons to whom the Software is
+// furnished to do so, subject to the following conditions :
+//
+// The above copyright notice and this permission notice shall be included in
+// all copies or substantial portions of the Software.
+//
+// THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+// IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+// FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+// AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+// LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+// OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
+// THE SOFTWARE.
+package com.microsoft.identity.common.internal.dto;
+
+import com.google.gson.annotations.SerializedName;
+
+import java.util.Calendar;
+import java.util.Date;
+import java.util.concurrent.TimeUnit;
+
+import static com.microsoft.identity.common.internal.dto.PrimaryRefreshTokenRecord.SerializedNames.*;
+
+public class PrimaryRefreshTokenRecord extends Credential {
+
+    public static class SerializedNames extends Credential.SerializedNames {
+        /**
+         * String of family id.
+         */
+        public static final String FAMILY_ID = "family_id";
+        /**
+         * String of realm.
+         */
+        public static final String REALM = "realm";
```

Comment:
This now appears unused... do we need a property to track the realm? If not, can this be removed?

It would be great to understand whether the PRT needs to be associated with a `realm`
- Fundamentally, it has a `realm` component of its cache key, but its always `""` -- can we safely assume we will never need to lookup a PRT by `realm`?

Reply:
From[ this doc](https://identitydivision.visualstudio.com/DevEx/_git/AuthLibrariesApiReview?path=%2FSSO%2FSchema.md&_a=preview&anchor=primary-refresh-token-schema), realm is not needed in PRT. I'll delete this field.

Reply:
Deleted